core/src/main/java/org/web3j/protocol/rx/Web3jRx.java

```diff
@@ -23,96 +21,98 @@
      * Create an observable to filter for specific log events on the blockchain.
      *
      * @param ethFilter filter criteria
-     * @return Observable that emits all Log events matching the filter
+     * @return a @{@link Flowable} instance that emits all Log events matching the filter
      */
-    Observable<Log> ethLogObservable(EthFilter ethFilter);
+    Flowable<Log> ethLogObservable(EthFilter ethFilter);
 
     /**
      * Create an Observable to emit block hashes.
      *
-     * @return Observable that emits all new block hashes as new blocks are created on the
-     *         blockchain
+     * @return a @{@link Flowable} instance that emits all new block hashes as new blocks are
+     *      created on the blockchain
      */
-    Observable<String> ethBlockHashObservable();
+    Flowable<String> ethBlockHashObservable();
 
     /**
      * Create an Observable to emit pending transactions, i.e. those transactions that have been
      * submitted by a node, but don't yet form part of a block (haven't been mined yet).
      *
-     * @return Observable to emit pending transaction hashes.
+     * @return a @{@link Flowable} instance to emit pending transaction hashes.
```

Comment:
I believe you only need to say `{@link Flowable}`, i.e. you can remove the initial `@`.